nebula/client.go

```diff
@@ -0,0 +1,458 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+package nebula
```

Comment:
I think ngdb (nebula graph database) may be more suitable than nebula as a package name, because names are not easy to conflict.

Reply:
just same as the folder name.